packages/ui-react/package.json

```diff
@@ -45,7 +45,8 @@
     "typescript-plugin-css-modules": "^5.0.2",
     "vi-fetch": "^0.8.0",
     "vite-plugin-svgr": "^4.2.0",
-    "vitest": "^1.3.1"
+    "vitest": "^1.3.1",
+    "wicg-inert": "^3.1.2"
```

Comment:
This is used for web but is also needed for vitest because setting `inert = true` does nothing without this polyfill.